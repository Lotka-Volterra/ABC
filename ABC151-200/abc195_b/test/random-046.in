19912024 601607155 627646101
