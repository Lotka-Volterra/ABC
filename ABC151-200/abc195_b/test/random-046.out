1044 31520
